pages/news.html

```diff
@@ -95,7 +95,7 @@
 
   <body>
     <ul class="list-group list-group-flush news-list">
-      <li class="list-group-item"><b>[January 22nd, 2025]</b> <b>Announcement:</b> We are happy to announce the <a href="https://posydon.org/summer_school.html">first POSYDON summer school</a> in September 2025.</li>
+      <li class="list-group-item"><b>[January 22nd, 2025]</b> <b>Announcement:</b> We are happy to announce the <a href="https://posydon.org/school.html">first POSYDON school</a> in September 2025.</li>
```

Comment:
I'd suggest to although add a new entry to announce the opening of the registration!
```suggestion
      <li class="list-group-item"><b>[April 7th, 2025]</b> <b>Announcement:</b> The <a href="https://posydon.org/posydon-school-2025/#apply">registration</a> to the POSYDON school is now open. More details can be found on the <a href="https://posydon.org/posydon-school-2025">2025 school web page</a>.</li>
      <li class="list-group-item"><b>[January 22nd, 2025]</b> <b>Announcement:</b> We are happy to announce the <a href="https://posydon.org/school.html">first POSYDON school</a> in September 2025.</li>
```